seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -1038,7 +1045,7 @@
         ""
       ],
       "tail": [
-        "if(typeof myFirstName !== \"undefined\" && typeof myLastName !== \"undefined\"){(function(){return myFirstName + ', ' + myLastName;})();}"
+        "if(typeof myFirstName !== \"undefined\" && typeof myLastName !== \"undefined\"){ console.log('myFirstName + ', ' + myLastName'); }"
```

Comment:
Ah, seems like the reason for this not working is obvious, the extra quotes shouldn't be there.